workloads/templates/common-templates.md

```diff
@@ -131,3 +131,46 @@ oc process rhel7-generic-tiny PVCNAME=mydisk NAME=rheltinyvm | oc apply -f -
 
 Please note that, after the generation step, VM objects and template objects have no relationship with each other besides the aforementioned label (e.g. changes
 in templates do not automatically affect VMs, or vice versa).
+
+## common template customization
+
+### template customization - memory, CPU
+
+There are three options to customize VM memory and CPU:
+
+ * select flavor - tiny, small, medium, etc. Each flavor grants to VM different amount of RAM and CPU cores.
+ * setting up directly editable fields spec.template.spec.domain.cpu.cores and spec.template.spec.domain.resources.requests.memory, example:
+
+```bash
+oc patch virtualmachine testguest --type merge -p '{"spec":{"template":{"spec":{"domain":{"cpu":{"cores":'3'}}}}}}'
+virtualmachine.kubevirt.io/testguest patched
+```
+ * use WebUI: WebUI supports different flavors and Workload Profiles. Also it supports custom amount of memory and CPUs
+
+### template customization - networking
+
+There are two options to customize Networking in VM:
+ * setting up directly editable field(see example above): spec.template.spec.networks. You can edit following:
+   * name (of interface)
+   * pod
+ * edit networking in webUI. Please note: WebUI also edits spec.domain.interfaces and supports changing mac address
+
+### template customization - disks
+
+There are two options to customize disks:
+
+ * Setting up sole variable - name of PersistentVolumeClaim(PVC)
```

Comment:
Again, you are describing customization of a VM and not the customization of the template here. There is no way to set variable in the template.